study-builder/studies/testboston/i18n/ar.conf

```diff
@@ -157,7 +157,8 @@
   },
   "baseline_symptom": {
     "name": "استبيان الأعراض",
-    "title": "<span>تحري أعراض الإصابة بفيروس Covid-19، <span class=\"activity-header__text\">يُرجى إخبارنا بما كنت تشعر به في الوقت الذي أخذت فيه عينة اختبار الكشف عن فيروس COVID-19</span></span>"
+    "title": "<span>تحري أعراض الإصابة بفيروس Covid-19، <span class=\"activity-header__text\">يُرجى إخبارنا بما كنت تشعر به في الوقت الذي أخذت فيه عينة اختبار الكشف عن فيروس COVID-19</span></span>",
+    "sample_collect_msg": ""
```

Comment:
Since we don't have translations for other languages, they will be blank.

Reply:
Thanks for illustrating this--it's super helpful for the small update I'm also in the middle of making.